tools/check_jsonnet

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+success=1
+JSONNETS=$(git ls-files schedule/ test_data/ | grep '.json\?net$' | xargs echo)
+
+if test -n "$JSONNETS"; then
```

Comment:
could you replace "jsonnets" by "jsonnet_files" ? that plural works poorly.

Reply:
updated